apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: VideoPlayer
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/VideoPlayer/VideoPlayer.tsx
+storybook: '/brand/storybook/?path=/story/components-videoplayer--playground'
+description: Use the video player component to present videos availible via CDN or local.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {VideoPlayer} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<VideoPlayer title="HLS On Demand Demo" crossOrigin="anonymous">
+  <VideoPlayer.Source src="https://cdn.api.video/vod/vi36GGvd6PoTqViQLxBWwHjJ/mp4/1080/source.mp4" />
+  <VideoPlayer.Track
+    src="https://raw.githubusercontent.com/JoshBowdenConcepts/Subtitles-Example/main/sample.vtt"
+    default
+    kind="subtitles"
+    srcLang="en"
+    label="English"
+  />
+</VideoPlayer>
+```
+
+## With poster
+
+```jsx live
+<VideoPlayer
+  poster="https://github.githubassets.com/images/icons/media-player/poster.jpg"
+  title="HLS On Demand Demo"
+  crossOrigin="anonymous"
+>
+  <VideoPlayer.Source src="https://cdn.api.video/vod/vi36GGvd6PoTqViQLxBWwHjJ/mp4/1080/source.mp4" />
+  <VideoPlayer.Track
+    src="https://raw.githubusercontent.com/JoshBowdenConcepts/Subtitles-Example/main/sample.vtt"
+    default
+    kind="subtitles"
+    srcLang="en"
+    label="English"
+  />
+</VideoPlayer>
+```
+
+## Without branding
+
+```jsx live
+<VideoPlayer
+  branding={false}
+  title="HLS On Demand Demo"
+  crossOrigin="anonymous"
+>
+  <VideoPlayer.Source src="https://cdn.api.video/vod/vi36GGvd6PoTqViQLxBWwHjJ/mp4/1080/source.mp4" />
+  <VideoPlayer.Track
+    src="https://raw.githubusercontent.com/JoshBowdenConcepts/Subtitles-Example/main/sample.vtt"
+    default
+    kind="subtitles"
+    srcLang="en"
+    label="English"
+  />
+</VideoPlayer>
+```
+
+## Component props
+
+<h3>
+  VideoPlayer <Label>Required</Label>
+</h3>
+
+`VideoPlayer` provides a React alternative to the native HTML `<video />`.
+
+The component API supports all standard HTML attribute props, while providing some additional behavior as described below.
+
+| Name        | Type                                   | Default | Required | Description                                                          |
+| :---------- | :------------------------------------- | :-----: | :------: | :------------------------------------------------------------------- |
+| `title`     | `boolean`                              |         |  `true`  | Sets the title of the video                                          |
```

Comment:
Should this be a `string` type instead of boolean?

```suggestion
| `title`     | `string`                              |         |  `true`  | Sets the title of the video                                          |
```

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/6f9a9c8740844ac6d50be622f1c9aab106c564dc)